TYPO3.Neos.Kickstarter/composer.json

```diff
@@ -5,12 +5,12 @@
     "license": "GPL-3.0+",
     "require": {
         "typo3/neos": "*",
-        "typo3/kickstart": "*",
+        "typo3/kickstart": "~3.3.0",
         "typo3/neos-nodetypes": "*"
```

Comment:
We can add it, in fact I think the kickstarter doesn'T really care about node types almost to a point where we could drop the dependency, but I didn't check that. I for now just wanted to create a working state. Also I tried to avoid to add constraints on dependencies that are in the dev collection, but merely outside packages.